test_brain_friendly.py

```diff
@@ -186,63 +187,43 @@ def test_invalid_braces():
 
 # Full program testing
 def test_hello_world():
-    program = ('++++++++[>++++[>++>+++>+++>+<<<<-]>+>+>->>+[<]<-]'
-               '>>.>---.+++++++..+++.>>.<-.<.+++.------.--------.>>+.>++.')
-    output = io.BytesIO()
-    eval_program(program, 0, [0]*100, output=output)
-    assert output.getvalue() == b'Hello World!\n'
-
-    program = ('>++++++++[-<+++++++++>]<.>>+>-[+]++>++>+++[>[->+++<<+++>]<<]'
-               '>-----.>->+++..+++.>-.<<+[>[+>+]>>]<--------------.>>.+++.'
-               '------.--------.>+.>+.')
-    output = io.BytesIO()
-    eval_program(program, 0, [0]*100, output=output)
-    assert output.getvalue() == b'Hello World!\n'
+    prefix = 'test_programs/hello_world/hello_world'
+    suffixes = ['_commented.bf', '_no_comma.bf',
+    '_no_comma2.bf', '_lowercase.bf', '_no_newline.bf']
+    filenames = [prefix+suffix for suffix in suffixes]
+    # Not all of these are the same.
+    file_outputs = [b'Hello World!\n', b'Hello World!\n',
```

Comment:
I feel icky about separating the outputs from the files like this, it seems like a failure of the separation of concerns.